pkg/ibmvpc/parser.go

```diff
@@ -30,6 +30,16 @@ func ParseResourcesFromFile(fileName string) (*datamodel.ResourcesContainerModel
 	return &config, nil
 }
 
+func getRegionByName(regionName string, regionToStructMap map[string]*Region) *Region {
```

Comment:
if `regionName` is empty string, return `nil`? @haim-kermany  please confirm

Reply:
Please return a Region with empty name, not nil.
Or there should be a default name, @adisos ?

Reply:
that is what we do right now.. if regionName is empty string we return region struct with empty name ""
no there is no default name